src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -101,30 +98,37 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
           </Box>
         )}
 
-        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'start' }}>
+        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'space-evenly' }}>
           <Button
+            sx={{
+              borderRadius: '15px 0px 15px 0px',
+              bgcolor: 'white',
+              border: '2px solid #4A7AC1',
+              borderRight: '5px solid #4A7AC1',
+              borderLeft: '5px solid #4A7AC1',
```

Comment:
Stusser jo litt over hvorfor denne knappen skal være så ulikt alt annet, men la gå. Beste jeg klarer for å unngå duplisering:
```suggestion
              bgcolor: 'white',
              borderStyle: 'solid',
              borderRadius: '8px 0px',
              borderWidth: '2px 5px',
              borderColor: 'generalSupportCase.main',
```

fra figma:
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/3f2c3c0d-ca3b-40cd-9c3a-90bb0a423528)